python/tvm/contrib/debugger/debug_executor.py

```diff
@@ -281,6 +282,42 @@ def run_individual(self, number, repeat=1, min_repeat_ms=0):
         ret = self._run_individual(number, repeat, min_repeat_ms)
         return ret.strip(",").split(",") if ret else []
 
+    def run_individual_node(self, index, number, repeat=1, min_repeat_ms=0):
+        """Benchmark a single node in the serialized graph.
```

Comment:
Can you specify that this does not do any data transfer and uses arrays that are already on the device?

Reply:
Done